src/components/Topbar/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -17,12 +17,10 @@ export const StyledTopbarBase = styled(TopbarBase)`
   }
 `
 
-export const DrawerButton = styled(Button)<CommonStudioTopbarProps>`
+export const DrawerButton = styled(Button)`
   svg {
     width: auto;
     height: auto;
```

Comment:
Btw, why is this needed? The `Button` component supports icons, right? So my guess would be we don't need to size them ourselves and they just work out of the box

Reply:
It was misaligned without it, but you are right - it should work out of the box. Had to replace chevron icon with the one from the Mystery Box 😃 Fixed now